wsimod/nodes/land.py

```diff
@@ -351,6 +376,44 @@ def __init__(
         self.processes = []
         self.outflows = []
 
+    def apply_overrides(self, overrides = Dict[str, Any]):
+        """Override parameters.
+
+        Enables a user to override any of the following parameters: 
+        area and depth (both will update the capacity), pollutant_load (the
+        entire dict does not need to be redefined, only changed values need to
+        be included).
+
+        Args:
+            overrides (Dict[str, Any]): Dict describing which parameters should
+                be overridden (keys) and new values (values). Defaults to {}.
+        """
+        self.surface = overrides.pop("surface", 
+                                  self.surface)
+        self.pollutant_load.update(overrides.pop("pollutant_load", {}))
+        
+        self.area = overrides.pop("area", 
+                                  self.area)
+        self.depth = overrides.pop("depth", 
+                                  self.depth)
+        self.capacity = self.area * self.depth
+        
+        if 'capacity' in overrides.keys():
+            overrides.pop('capacity')
+            print('ERROR: specifying capacity is depreciated in overrides for surface, please specify depth and area instead')
+            
+        # overrides data_input_dict
+        from wsimod.orchestration.model import read_csv
+        content = overrides.pop('data_input_dict', self.data_input_dict)
+        if isinstance(content, str):
+            self.data_input_dict = read_csv(content)
+        elif isinstance(content, dict):
+            self.data_input_dict = content
+        else:
+            print('ERROR: not recognised format for data_input_dict')

```

Comment:
```suggestion
            raise ValueError(f'{content.__class__} is not a recognised format for data_input_dict')
```

Reply:
and similar for other errors. We can use print for a warning, but probably not an error